medium-tests/src/test/java/mediumtest/tracking/IssueTrackingMediumTests.java

```diff
@@ -667,10 +666,45 @@ void it_should_match_a_previously_tracked_issue_with_a_server_issue_when_binding
       .containsExactly(firstPublishedIssue.getId(), "key", serverIssueIntroductionDate);
   }
 
+  @Test
+  void it_should_stream_issue(@TempDir Path baseDir) throws IOException, GitAPIException {
```

Comment:
Reminder.
Here we don't test accumulation and raising sequence of issues that arrive one after another. Maybe we should write such test later?

Reply:
Good point, I changed the Sensor test to something as follows:
> raiseIssue();
    pause(500);
    raiseIssue();
    pause(500);

This lets me verify that there are 2 intermediate issues raised